src/main/java/ml/shifu/shifu/core/VariableSelector.java

```diff
@@ -176,8 +177,13 @@ public String toString() {
 
         List<Tuple> newParetoList = sortByPareto(paretoList);
 
-        int expectedVarNum = Math.min(ksList.size(), this.modelConfig.getVarSelectFilterNum());
-
+        int expectedVarNum = Math.min(ksList.size(), modelConfig.getVarSelectFilterNum());
+        log.info("expectedNum:"+expectedVarNum);
```

Comment:
SLF4j, better this style
log.info("expectedNum: {}", expectedVarNum);

For info level, please make it a full sentence like or set it to debug level